src/components/views/elements/StyledPollCheckbox.tsx

```diff
@@ -0,0 +1,65 @@
+/*
```

Comment:
Can't we use / extend https://github.com/matrix-org/matrix-react-sdk/blob/develop/src/components/views/elements/StyledCheckbox.tsx for the use in polls?